### ISSUE TITLE
Missing changelog between 4.3 and 5.1

### ISSUE THREAD

**Issue by ZeeCoder** [Olical/EventEmitter#124]
Hi!

First of all thanks for creating this package, we use it in production to great results.

We currently use 4.3, but I see the latest version is 5.1.

That, to me, implies a breaking change in the public API, but cannot figure out what changed exactly and so can't blindly update.

Could someone sum up the changes made between the two versions?

**Comment by Olical** [comment]
I can't quite remember, but from having a look through the delta https://github.com/Olical/EventEmitter/compare/v4.3.0...v5.1.0 I don't think much changed. I think the big thing was looping over events in a normal manner, instead of backwards. I think `while (i--)` used to be faster in some browsers when I originally wrote this? I fell for jsperf microbenchmarks in those days :(

I can't really see anything else, I'm sorry I forgot to document it.

In this unreleased master diff https://github.com/Olical/EventEmitter/compare/v4.3.0...master there is an extra type check, but that should be fine. Honestly hardly anything changed. I think I bumped major because I changed the execution order. This *could* change program behavior, although I'd hope not!
